### PR TITLE
Add label to the blue track representing the full protein in EntityViewer

### PR DESCRIPTION
**Pull request by azangru**
## Type
- Improvement to existing feature

## Related JIRA Issue(s)
https://www.ebi.ac.uk/panda/jira/browse/ENSWBSITES-786

## Description
Adding a label to the previously unlabelled protein track in EntityViewer (protein view):

image

## Deployment URL
http://protein-track-label.review.ensembl.org/

## Views affected
EntityViewer (protein view)

### CODE REVIEW

File: src/ensembl/src/content/app/entity-viewer/gene-view/components/protein-image/ProteinImage.tsx [hunk 1]
```diff
@@ -42,7 +42,8 @@ const ProteinImage = (props: ProteinImageProps) => {
     .range([0, props.width])
     .clamp(true);
 
-  const midStyles = classNames(transcriptsListStyles.middle, styles.middle);
+  const midStyles = transcriptsListStyles.middle;
+  const rightStyles = classNames(transcriptsListStyles.right, styles.label);
```
Comment (msmanoj) [inline src/ensembl/src/content/app/entity-viewer/gene-view/components/protein-image/ProteinImage.tsx:46]: Would it be better to rename these to something like `trackContainerStyles` and `labelStyles`?

Comment (azangru) [reply src/ensembl/src/content/app/entity-viewer/gene-view/components/protein-image/ProteinImage.tsx]: I don't feel strongly one way or the other. Renamed them if you prefer these names.

**Comment by msmanoj** [comment]
Just wondering what is the purpose of this label here?  It feels more like we should be displaying some value here.

**Comment by azangru** [comment]
Andrea just wanted to give a name to what this track represents 😃 

For what it's worth, I also think that "amino acid length" or "protein length" is not a proper label, because this track represents the protein itself rather than its length. But it's up to Andrea, Andy or @ens-st3 to decide on the exact wording of this label if they want to change it 🤷‍♂️

**Pull request merged**